packages/dicom/package.json

```diff
@@ -28,7 +28,7 @@
     "build:python:versionSync": "itk-wasm pnpm-script build:python:versionSync",
     "publish:python": "itk-wasm pnpm-script publish:python",
     "test": "pnpm test:data:download && pnpm build:gen:python && pnpm test:python",
-    "test:data:download": "dam download test/data test/data.tar.gz bafybeicskxufnvuem6342pkfwgeo3siiozgzmfo5f34woge6aptuzuwzzu https://github.com/InsightSoftwareConsortium/ITK-Wasm/releases/download/itk-wasm-v1.0.0-b.119/dicom-test-data.tar.gz https://w3s.link/ipfs/bafybeiby67winzvozowf4moqthwunuxxscssitnb6wahxv4ugvfxhu2vki/data.tar.gz",
+    "test:data:download": "dam download test/data test/data.tar.gz bafybeifswmu56lqciufqydkgjhcjbamivextuqylpjdh672ivnrmispgfy https://data.kitware.com/api/v1/file/66be71dcf87a980650f422a1/download",
```

Comment:
after merge, please also upload to the latest GitHub published Release and create a new PR adding the URL

Reply:
tracking in issue #1202